tools/azure-sdk-tools/packaging_tools/templates/README.rst

```diff
@@ -30,4 +30,4 @@ If you encounter any bugs or have suggestions, please file an issue in the
 section of the project.
 
 
-.. image::  https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-python%2F{{package_name}}%2FREADME.png
+.. image::  https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-python%2Ftools%2Fazure-sdk-tools%2Fpackaging_tools%2Ftemplates%2FREADME.png
```

Comment:
This one is a template, you want to keep `{{package_name}}`` in it.

Reply:
I will fix this.